project-base/tests/App/Functional/EntityExtension/EntityExtensionTest.php

```diff
@@ -4,17 +4,17 @@
 
 namespace Tests\App\Functional\EntityExtension;
 
-use App\Model\Category\Category;
-use App\Model\Order\Order;
 use App\Model\Order\Order as ExtendedOrder;
 use App\Model\Product\Brand\Brand as ExtendedBrand;
-use App\Model\Product\Product;
 use Doctrine\Common\Annotations\AnnotationReader;
 use Doctrine\Common\Persistence\Mapping\Driver\MappingDriverChain;
 use Doctrine\ORM\Mapping\Driver\AnnotationDriver;
 use Doctrine\ORM\Tools\SchemaTool;
+use Shopsys\FrameworkBundle\Model\Category\Category;
```

Comment:
I think some explanation in commit message would be nice. Something like: entity extensibility is expected to be only two levels - framework entity and then project-base entity. Multi level extensibility may cause problems and is not supported

Reply:
Ok, I changed it.